code/modules/projectiles/projectile/magic.dm

```diff
@@ -288,7 +288,7 @@
 	for(var/obj/item/W in contents)
 		new_mob.equip_to_appropriate_slot(W)
 
-	M.log_message("became [new_mob.real_name]", LOG_ATTACK, color="orange")
+	M.log_message("became [new_mob.name]([new_mob.type])", LOG_ATTACK, color="orange")
```

Comment:
`M` ends up being deleted at the end of this proc, will this log message still be accessible?

Reply:
Yes it is still transferred to the new mob like is always has. I only discovered the blank logging by checking after polymorphing myself from a human into a xeno.